src/components/Footer/FooterStyle.js

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+
+const StyledFooter = styled.div`
+    display: flex;
+    justify-content: center;
```

Comment:
The rows in the footer aren't aligned with each other. I think it's because of the `justify-content: center`

I would suggest using the code from figma here:
`display: flex;
flex-direction: row;
justify-content: space-between;
align-items: flex-start;
padding: 50px 0px 30px;`

However, doing this you will also need to get the CSS from figma for the logo and GitHub images.